package.json

```diff
@@ -112,8 +44,12 @@
     "gcm"
   ],
   "main": "index",
+  "files": [
+    "index.js",
+    "lib/"
+  ],
   "scripts": {
-    "test": "sh node_modules/.bin/mocha -R spec test/**/*Spec.js"
+    "test": "mocha test/**/*Spec.js"
```

Comment:
Unfortunately this doesn't run on Windows per default. I am wondering if there is an obvious cross-platform solution for this ...